docs/guides/on-demand.mdx

```diff
@@ -67,17 +67,17 @@ the response will return the `id` of that asset
 
 The response will indicate the status of the uploaded assets as status:
 
-- `waiting` or `status`: "ready".
+- `status: "waiting"` or `status: "ready"`.
 
-- If status: "waiting" is returned in the response, the asset is not available:
+- If `status: "waiting"` is returned in the response, the asset is not available:
 
-  - make the API call again until status: "ready".
+  - make the API call again until `status: "ready"`.
 
-- If the status indicates that it is "ready", it is available.
+- If the status indicates that it is `"ready"`, it is available.
```

Comment:
We should update these instructing users to use `status.phase` instead of just `status`, since that has changed in the API.